cli/fossilize_replay.cpp

```diff
@@ -419,6 +427,115 @@ struct ThreadedReplayer : StateCreatorInterface
 		return true;
 	}
 
+	void get_pipeline_stats(ResourceTag tag, Hash hash, VkPipeline pipeline)
+	{
+		VkPipelineInfoKHR pipeline_info;
```

Comment:
`pipeline_info = { VK_STRUCTURE_TYPE ... }` is used throughout the codebase to init structs.